arch/configure.defaults

```diff
@@ -1597,8 +1597,7 @@ RLFLAGS		=
 CC_TOOLS        =       $(SCC) 
 
 LIB_EXTERNAL    = \
-                     ../external/io_netcdf/libwrfio_nf.a CONFIGURE_NETCDF_PATH/lib/libnetcdf.lib \
-                     ../external/wavelet/libWavelet.a ../external/wavelet/lib_wavelet.a
```

Comment:
I couldn't find any way to generate an `external/wavelet/libWavelet.a`, so I removed that link here and did not add it for the Cygwin link line.  If that archive was moved somewhere rather than removed, I should change these to point to the new location.

Reply:
This wavelet library looks to be part of the data assimilation. It is located in var/external.